readme.md

```diff
@@ -1,16 +1,20 @@
-## COMP 4350 - [Group Name]
+# COMP 4350 - Nerdy Nomads
```

Comment:
Should we add our course section ???

Also, eventually, we would have to change the title to our project name after we decide.